frontline-functions-quickstart/functions/callbacks/twilio-conversations.protected.js

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable camelcase */
+const assets = Runtime.getAssets();
+
+const { getCustomerByNumber } = require(assets['/providers/customers.js'].path);
+
+const setCustomerParticipantProperties = async (
+  customerParticipant,
+  customerDetails
+) => {
+  console.log('[setCustomerParticipantProperties] ', {
+    customerParticipant,
+    customerDetails,
+  });
+  const participantAttributes = JSON.parse(customerParticipant.attributes);
+  const customerProperties = {
+    attributes: JSON.stringify({
+      ...participantAttributes,
+      avatar: participantAttributes.avatar || customerDetails.avatar,
+      customer_id:
+        participantAttributes.customer_id || customerDetails.customer_id,
+      display_name:
+        participantAttributes.display_name || customerDetails.display_name,
+    }),
+  };
+
+  // If there is difference, update participant
+  if (customerParticipant.attributes !== customerProperties.attributes) {
+    // Update attributes of customer to include customer_id
+    await customerParticipant
+      .update(customerProperties)
+      .catch((e) => console.log('Update customer participant failed: ', e));
+  }
+};
+
+// eslint-disable-next-line consistent-return
+exports.handler = async function (context, event, callback) {
+  console.log('((( Conversations Callback )))) ');
+  console.log('CONTEXT: ', JSON.stringify(context));
```

Comment:
Same here. Avoid both logging the entire `context` and `event` object as they might contain sensitive data. I assume this was for debugging.